app/src/main/java/com/taskruler/MainActivity.kt

```diff
@@ -165,7 +163,7 @@ fun UserTasksList(
                 Text(text = userTaskText, fontSize = 18.sp, modifier = Modifier.padding(end = 8.dp))
                 Icon(imageVector = Icons.Filled.ArrowDropDown, contentDescription = "")
                 DropdownMenu(expanded = expanded, onDismissRequest = { expanded = false }) {
-                    userTasks.forEach(){
+                    userTasks.forEach{
```

Comment:
Removed unnecessary parenthesis from function call with lambda
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/for-each.html